msgraph-calendar-demo/webContent/layouts/frame-10.xhtml

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
```

Comment:
thanks for migrating to frame-10.xhtml good refurbishment; can you please check for no longer required files in the webContent, and remove the stuff no longer required. At least the frame-9.xhtml should be ready to be removed :recycle: